### DISCUSSION TITLE
How to remove the default left column in DetailList Control

### DISCUSSION THREAD

**Discussion by AstroNorman** [microsoft/powercat-creator-kit#59] [Q&A] [answered]
<img width="287" alt="column" src="https://user-images.githubusercontent.com/106515086/177932296-aaa9fef0-b5ff-4ba4-a4ca-c4161b3b1ead.png">

How do I remove the default left column seen in the picture above?

**Comment by Ramakrishnan24689** [comment, marked as answer]
HI @AstroNorman,

Defining **Selection Type** property to **None** will remove the selection column.

<img width="141" alt="image" src="https://user-images.githubusercontent.com/11135056/177938365-23105546-aa3e-4222-9f7d-84343df02153.png">

**Reply by AstroNorman** [reply]
Thanks @Ramakrishnan24689
